.github/workflows/update-getting-started-guide.yml

```diff
@@ -0,0 +1,56 @@
+name: Update getting started guide
+
+on:
+  release:
+    types: ['released']
+  pull_request:
+    branches: ['main', 'release/**']
+    paths:
+      - '.github/workflows/update-getting-started-guide.yml'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-20.04
+    env:
+      RELEASE_TAG: ''
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Mock release tag on pull request
+        if: github.event_name == 'pull_request'
+        run: echo "RELEASE_TAG=v0.0.0-${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+
+      - name: Set published release tag
+        if: github.event_name == 'release'
+        run: echo "RELEASE_TAG=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+
+      - name: Update getting started version
+        run: |
+          release_tag=${{ env.RELEASE_TAG }}
+          release_version=${release_tag/v/}
+          sed -i -E "s/version=\"([0-9]+\.){2}[0-9]+\"/version=\"${release_version}\"/" docs/getting-started.md
+          git diff
+
+      - name: Create PR
+        if: github.event_name == 'release'
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: 'Update getting started to ${{ github.event.release.tag_name }}'
+          commit-message: 'Update getting started to ${{ github.event.release.tag_name }}'
+          body: |
+            This PR must be closed and reopened manually to trigger automated checks.
+
+            Auto-generated by [create-pull-request][1].
```

Comment:
Can't this just be markdown? Or is this going to be added as a commit message upon merge as well?

Reply:
Yeah, this can be markdown. It is not included in the commit message.